csrc/id_model/id_model.cpp

```diff
@@ -554,8 +553,8 @@ StatefulInliningInfo buildStatefulInliningInfo(
         auto consumer_tv_i = consumer_tvs.vector().at(i);
         auto all_consumer_i_ids = ir_utils::allIDsOf(consumer_tv_i);
 
-        auto sibling_map =
-            exact_graph.buildMapBetween(all_consumer_ids, all_consumer_i_ids);
+        auto sibling_map = permissive_graph.buildMapBetween(
```

Comment:
I don't think exact graph and permissive graph makes any difference in siblings.